redis/src/types.rs

```diff
@@ -104,9 +104,8 @@ pub enum ErrorKind {
     /// Error Serializing a struct to JSON form
     Serialize,
 }
-
 /// Internal low-level redis value enum.
-#[derive(PartialEq, Eq, Clone)]
+#[derive(PartialEq, Clone)]
```

Comment:
I think we should just remove it for now. I'm not very familiar with `float_eq` and am not sure what benefit it would provide for us at this point. We can add later if we have a use case.